hack/local-up-cluster.sh

```diff
@@ -653,6 +654,7 @@ function start_controller_manager {
     fi
 
     cloud_config_arg=("--cloud-provider=${CLOUD_PROVIDER}" "--cloud-config=${CLOUD_CONFIG}")
+    cloud_config_arg+=("--configure-cloud-routes=${CONFIGURE_CLOUD_ROUTES}")
```

Comment:
@nckturner is `true` the current default if `--configure-cloud-routes` is not specified in the command line?

Reply:
Yes, that is correct.